pkg/apis/pipeline/v1/pipeline_validation.go

```diff
@@ -497,6 +497,21 @@ func (pt *PipelineTask) extractAllParams() Params {
 	return allParams
 }
 
+func (pt *PipelineTask) GetVarSubstitutionExpressions() ([]string, bool) {
+	var allExpressions []string
+	if pt.TaskSpec != nil {
+		for _, step := range pt.TaskSpec.Steps {
+			stepExpressions, _ := step.GetVarSubstitutionExpressions()
```

Comment:
Could you help me to understand why is the bool return value needed here? It seems that this is implying whether the expressions is of len 0, but the actual returned values are unused.

Reply:
the returned bool value is indeed not used. I modified the function and only returned expressions.